app/src/androidTest/java/au/com/shiftyjelly/pocketcasts/models/db/PodcastManagerTest.kt

```diff
@@ -42,7 +44,15 @@ class PodcastManagerTest {
 
         val episodeManager = mock<EpisodeManager>()
         val playlistManager = mock<PlaylistManager>()
-        val settings = mock<Settings>()
+
+        val settings = SettingsImpl(
+            sharedPreferences = PreferenceManager.getDefaultSharedPreferences(context),
+            privatePreferences = PreferenceManager.getDefaultSharedPreferences(context),
+            context = context,
+            firebaseRemoteConfig = mock(),
+            moshi = Moshi.Builder().build(),
+        )
+
```

Comment:
I don't feel great about this, but no matter what way I tried to mock out the settings, I got crashes in the tests when it tried to retrieve a `UserSetting::flow` object. Let me know if you have any ideas for getting this to work with mocks.

Reply:
I found a way to avoid this in https://github.com/Automattic/pocket-casts-android/pull/1234/commits/2767f55d32961a2516b758513b0fcb73e0bc120e